chromium_src/components/sync/engine_impl/commit.cc

```diff
@@ -146,6 +169,10 @@ brave_sync::RecordsListPtr ConvertCommitsToBraveRecords(
       bookmark->metaInfo.push_back(metaInfo);
 
       record->SetBookmark(std::move(bookmark));
+
+      if (entity.parent_id_string() == kOtherBookmarksTag) {
+        record_list->push_back(CreateOtherBookmarksRecord(record.get()));
```

Comment:
I don't understand, why `other bookmarks` record is created each time when we meet an entity whom parent is `other bookmarks`? Just to make create it one time after when we finished `for` loop if we met any child of `other bookmark`.
I guess there is a reason for this, but have no idea why for.

Reply:
you are right, thanks. it's fixed.